commerce-frontend-js/src/main/resources/META-INF/resources/js/gallery/_gallery.scss

```diff
@@ -0,0 +1,89 @@
+.product-gallery {
+  .card img {
+    max-height: 100%;
+  }
+
+  .main-image {
+    cursor: zoom-in;
+  }
+
+  .gallery-overlay {
+    background: #fff;
+    cursor: zoom-out;
+    height: 100vh;
+    left: 0;
+    position: fixed;
+    top: 0;
+    width: 100vw;
+    z-index: 10000;
+
+    img {
+      left: 50%;
+      max-height: 100%;
+      max-width: 100%;
+      position: absolute;
+      top: 50%;
+      transform: translate(-50%, -50%);
+    }
+  }
+
+  .gallery-thumbnails {
+    display: flex;
+    overflow-x: auto;
+
+    .card {
+      width: 75px;
+      flex-shrink: 0;
+
+      .aspect-ratio {
+        padding-bottom: 75%;
+      }
+    }
+    
+    .card + .card {
+      margin-left: 10px;
+    }
+  }
+
+  .prev,
+  .next {
+    background: linear-gradient(to right, rgba(0,0,0,0.4), transparent 50%);
+    background-position: 99%;
+    background-size: 200%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    transition: all ease 100ms;
+    width: 20%;
+    
+    &:hover {
+      background-position: 0%;
+    }
+  }
+
+  .prev {
+    cursor: w-resize;
+    left: 0;
+  }
+
+  .next {
+    cursor: e-resize;
+    right: 0;
+    transform: rotate(180deg);
+  }
+
+  .loading-animation {
```

Comment:
selectors should be sorted alphabetically -- I think that would be:

    .gallery-overlay
    .gallery-thumbnails
    .loading-animation
    .main-image
    .next, .prev
    .next
    .prev